src/pages/hyunjaeKim/Login/Login.js

```diff
@@ -1,8 +1,80 @@
 import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import "./Login.scss";
 
 class LoginHyunJae extends Component {
+  state = {
+    email: "",
+    password: "",
+  };
+
+  saveData = data => {
+    localStorage.setItem("TOKEN", data.TOKEN);
+  };
+
+  handleLogin = () => {
+    fetch("http://10.58.0.45:8000/users/login", {
+      method: "POST",
+      body: JSON.stringify({
+        ...this.state,
+      }),
+    })
+      .then(res => res.json())
+      .then(data => {
+        if (data.TOKEN) {
+          this.saveData(data);
+          this.props.history.push("/main-jae");
+        } else {
+          alert("회원가입을 해주세요!");
+        }
+      });
+  };
+
+  handleIDInput = e => {
+    const { value } = e.target;
+    this.setState({ ...this.state, email: value });
+  };
+
+  handlePWInput = e => {
+    const { value } = e.target;
+    this.setState({ ...this.state, password: value });
+  };
```

Comment:
```suggestion
  handleInput = e => {
    const { name, value } = e.target;
    this.setState({ ...this.state, [name]: value });
  };
```
- 기능 : input 태그로부터 value를 넘겨 받아 setState
- 대상 : email, password state

코드를 살펴보면 두 메서드의 기능은 동일하나 업데이트 하는 대상이 달라짐에 따라 중복 로직을 추가로 작성하신 것 같습니다.
input 태그의 name 속성과 객체 계산된 속성명 문법을 사용하면 위와 같이 업데이트 로직에 관련된 메서드를 하나로 줄일 수 있습니다.

Reply:
넵! 로그인 백프론트 결합 시 날로 작업한 내역이 정리가 안되어있었네요ㅠㅠ! 제안해주신 방법으로 개선하였습니다✨